frontend/app/ui/subscriptions/list/controller.js

```diff
@@ -49,6 +52,39 @@ export default class SubscriptionsListController extends Controller {
       : this._projects;
   }
 
+  @action export() {
+    const headings = [
+      this.intl.t("page.subscriptions.list.table.project"),
+      this.intl.t("page.subscriptions.list.table.customer"),
+      this.intl.t("page.subscriptions.list.table.billing"),
+      this.intl.t("page.subscriptions.list.table.time-purchased"),
+      this.intl.t("page.subscriptions.list.table.time-spent"),
+      this.intl.t("page.subscriptions.list.table.time-total"),
+      this.intl.t("page.subscriptions.list.table.time-unconfirmed"),
+    ].join(",");
+
+    const lines = this.projects
+      .toArray()
```

Comment:
Is the `toArray` required here?

Reply:
Yes. I don't remember what exactly was wrong but did I try without before.